src/components/navbar.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Link, NavLink } from 'react-router-dom';
+
+function Navbar() {
+  return (
+    <div className="container">
+      <nav className="nav-bar">
+        <div className="nav-brand">
+          <Link to="/" className="nav-img"><img src="planet.png" alt="logo" /></Link>
+          <h2>Space Travelers&apos; Hub </h2>
+        </div>
+        <ul className="nav-links">
+          <li><NavLink to="/" className={({ isActive }) => (isActive ? 'active-link nav-link' : 'nav-link')}>Rockets</NavLink></li>
+          <li><NavLink to="/categories" className={({ isActive }) => (isActive ? 'active-link nav-link' : 'nav-link')}>Missions</NavLink></li>
+          <span>|</span>
+          <li><NavLink to="/categories" className={({ isActive }) => (isActive ? 'active-link nav-link' : 'nav-link')}>My Profile</NavLink></li>
+        </ul>
```

Comment:
- [ ] As shown in the image below the Rockets link is underlined when it is active but Missions and My profile links are active and underlined at the same time, kindly fix this by separating those two active links as required to indicate which section/page the user is currently on. 
![Screenshot 2023-03-08 135720](https://user-images.githubusercontent.com/87595063/223709057-85b0eb7d-0e2a-4547-b464-8d58693c4b3f.jpg)

![Screenshot 2023-03-08 ](https://user-images.githubusercontent.com/87595063/223709281-f637aa8f-ae59-4981-9318-402b511512ff.png)

Reply:
Thanks for your comment. As i taught this was required in the other task so I have fixed that in the fetch PR